features/sharding/distsql/handler/src/main/java/org/apache/shardingsphere/migration/distsql/handler/query/ShowMigrationJobStatusQueryResultSet.java

```diff
@@ -45,24 +45,19 @@ public final class ShowMigrationJobStatusQueryResultSet implements DatabaseDistS
     
     @Override
     public void init(final ShardingSphereDatabase database, final SQLStatement sqlStatement) {
-        long currentTimeMillis = System.currentTimeMillis();
         List<InventoryIncrementalJobItemInfo> jobItemInfos = JOB_API.getJobItemInfos(((ShowMigrationStatusStatement) sqlStatement).getJobId());
         data = jobItemInfos.stream().map(each -> {
+            MigrationJobItemInfo jobItemInfo = (MigrationJobItemInfo) each;
             Collection<Object> result = new LinkedList<>();
-            result.add(each.getShardingItem());
-            InventoryIncrementalJobItemProgress jobItemProgress = each.getJobItemProgress();
+            result.add(jobItemInfo.getShardingItem());
+            InventoryIncrementalJobItemProgress jobItemProgress = jobItemInfo.getJobItemProgress();
             if (null != jobItemProgress) {
                 result.add(jobItemProgress.getDataSourceName());
                 result.add(jobItemProgress.getStatus());
                 result.add(jobItemProgress.isActive() ? Boolean.TRUE.toString() : Boolean.FALSE.toString());
                 result.add(jobItemProgress.getProcessedRecordsCount());
-                result.add(jobItemProgress.getInventory().getInventoryFinishedPercentage());
-                String incrementalIdleSeconds = "";
-                if (jobItemProgress.getIncremental().getIncrementalLatestActiveTimeMillis() > 0) {
-                    long latestActiveTimeMillis = Math.max(each.getStartTimeMillis(), jobItemProgress.getIncremental().getIncrementalLatestActiveTimeMillis());
-                    incrementalIdleSeconds = String.valueOf(TimeUnit.MILLISECONDS.toSeconds(currentTimeMillis - latestActiveTimeMillis));
-                }
-                result.add(incrementalIdleSeconds);
+                result.add(jobItemInfo.getInventoryFinishedPercentage());
+                result.add(jobItemInfo.getIncrementalIdleSeconds());
```

Comment:
It's `null != jobItemProgress` in if condition, but it use `jobItemInfo` here, it's not clear enough as before